src/evict/evict_page.c

```diff
@@ -708,5 +708,9 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
       !__wt_page_is_modified(page) || LF_ISSET(WT_REC_HS | WT_REC_IN_MEMORY) ||
         WT_IS_METADATA(btree->dhandle));
 
+    /* Fail 0.1% of the time. */
```

Comment:
This fail point has caused a bug and it is fixed in WT-9751. if we are going to backport this failing point, we need to backport that ticket also.